packages/block-editor/src/components/block-mover/index.js

```diff
@@ -83,6 +88,7 @@ function BlockMover( { clientIds, hideDragHandle } ) {
 				<ToolbarItem>
 					{ ( itemProps ) => (
 						<BlockMoverUpButton
+							disabled={ isPrevBlockTemplatePart }
```

Comment:
I think `disabled` is a prop that we should avoid for accessibility reasons.

Reply:
but it is already used, in the component it's set to true if for example isFIrstBlock or isLastBlock